app/src/main/java/ch/epfl/culturequest/notifications/PushNotification.java

```diff
@@ -97,6 +101,7 @@ public Notification buildNotification(Context context) {
                 .setPriority(NotificationCompat.PRIORITY_HIGH)
                 .setDefaults(Notification.DEFAULT_ALL)
                 .setContentIntent(pendingIntent)
+                .setAutoCancel(true)
```

Comment:
Oh that's right we need to do that too! I was going to add it on one of my PR but thanks for doing it here!